classes/person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :age, :name
-  attr_reader :id, :parent_permission
+  attr_reader :id, :parent_permission, :rentals
```

Comment:
- [ ]  Kindly I recommend you to set `*Getters and Setters for @rentals` instead of setting only  `attr_reader for @rentals` because when you set up `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`, A person should have many rentals: So in `attr_accessor` provides both read and write access to an instance variable, while `attr_reader `only provides read access. So kindly Adjust like the below sample.

```
attr_accessor :name, :age, :rentals
 attr_reader :id, :parent_permission

```

Reply:
Done @Abenezer-Tilahun.